### PR TITLE
add straggler dbf-only factoids into ferc1 table metadata

### PR DESCRIPTION
**Pull request by cmgosnell**
# PR Overview
I started with just adding the dfb-only metadata fields into each table's metada doing it in a bespokey way. will probably parameterize this adding of these guys.

# PR Checklist

- [ ] Merge the most recent version of the branch you are merging into (probably `dev`).
- [ ] All CI checks are passing. [Run tests locally to debug failures](https://catalystcoop-pudl.readthedocs.io/en/latest/dev/testing.html#running-tests-with-tox)
- [ ] Make sure you've included good docstrings.
- [ ] For major data coverage & analysis changes, [run data validation tests](https://catalystcoop-pudl.readthedocs.io/en/latest/dev/testing.html#data-validation)
- [ ] Include unit tests for new functions and classes.
- [ ] Defensive data quality/sanity checks in analyses & data processing functions.
- [ ] Update the [release notes](https://catalystcoop-pudl.readthedocs.io/en/latest/release_notes.html) and reference reference the PR and related issues.
- [ ] Do your own explanatory review of the PR to help the reviewer understand what's going on and identify issues preemptively.

### CODE REVIEW

File: src/pudl/transform/ferc1.py [hunk 1]
```diff
@@ -5133,20 +5117,83 @@ class BalanceSheetLiabilitiesFerc1TableTransformer(Ferc1AbstractTableTransformer
...
+        """
+        tbl_meta = super().process_xbrl_metadata(xbrl_metadata_json)
+        facts_to_add = {
+            "xbrl_factoid": ["accumulated_deferred_income_taxes"],
+            "calculations": ["[]"],
+            "balance": ["credit"],
+            "ferc_account": [pd.NA],
+            "xbrl_factoid_original": ["accumulated_deferred_income_taxes"],
+            "intra_table_calc_flag": [True],
+            "row_type_xbrl": ["report_value"],
```
Comment (zaneselvans) [inline src/pudl/transform/ferc1.py:5134]: This should be "reported_value" in all of the fixes.

File: src/pudl/transform/ferc1.py [hunk 2]
```diff
@@ -2366,14 +2366,6 @@ def apply_xbrl_calculation_fixes(
...
                 ],
                 "unappropriated_undistributed_subsidiary_earnings": [
-                    {
-                        "calc_component_to_replace": {},
-                        "calc_component_new": {
-                            "name": "unappropriated_undistributed_subsidiary_earnings_previous_year",
-                            "weight": 1.0,
-                            "source_tables": ["retained_earnings_ferc1"],
-                        },
-                    },
```
Comment (zaneselvans) [inline src/pudl/transform/ferc1.py:2376]: Are these changes to the calculation fixes related to the dbf-only facts?

File: src/pudl/transform/ferc1.py [hunk 3]
```diff
@@ -5133,20 +5117,83 @@ class BalanceSheetLiabilitiesFerc1TableTransformer(Ferc1AbstractTableTransformer
...
+            {
+                "xbrl_factoid": dbf_onl_fact,
+                "calculations": "[]",
+                "balance": "credit",
+                "ferc_account": pd.NA,
+                "xbrl_factoid_original": dbf_onl_fact,
+                "intra_table_calc_flag": True,
+                "row_type_xbrl": "report_value",
+            }
+            for dbf_onl_fact in ["special_funds_all", "nuclear_fuel"]
```
Comment (zaneselvans) [inline src/pudl/transform/ferc1.py:5164]: The computer doesn't care but did you mean for this to be `dbf_only_fact`?

**Review by zaneselvans — CHANGES_REQUESTED** [review]

File: src/pudl/transform/ferc1.py (hunk 3 above)
Comment (cmgosnell) [reply src/pudl/transform/ferc1.py:5164]: lol yes

File: src/pudl/transform/ferc1.py (hunk 2 above)
Comment (cmgosnell) [reply src/pudl/transform/ferc1.py:2376]: ah no this is just one of the many instances of factoids that weren't actually in the data.

**Review by zaneselvans — APPROVED** [review]

**Review by zaneselvans — APPROVED** [review]

**Pull request merged**